packages/medusa/src/api/routes/store/variants/list-variants.ts

```diff
@@ -152,6 +144,7 @@ export default async (req, res) => {
     "limit",
     "offset",
     "expand",
+    "fields",
```

Comment:
Suggestion : depending on the previous one, should we use req.filterableFields?

Reply:
The `req.filterableFields` contains a mix of filterable fields for the ProductVariantService.list and price selection, so that's not possible without doing additional modifications.

Reply:
Something like we do usually :

const { something, ...filterableFields } = req.filterableFields 

Wdyt?

Reply:
Shuffled it around 👍 

It doesn't change the fact that `req.filterableFields` are not behaving 100% correctly in this case, which is what I was hinting. That said, I do like this better